src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/packageScanner.hpp

```diff
@@ -114,6 +117,9 @@ class PackageScanner final : public AbstractHandler<std::shared_ptr<ScanContext>
                                           versionStringLessThanOrEqual.c_str());
 
                                 data->m_elements[callbackData.cveId()->str()] = nlohmann::json::object();
+                                data->m_alerts[callbackData.cveId()->str()]["vulnerability"]["package"]["condition"] =
+                                    "Package less than or equal to " + versionString;
```

Comment:
You need to use versionStringLessThan or versionLessThanOrEqual

Ex

```suggestion
                                    versionStringLessThan.empty() ? "Package less than or equal to "  + versionStringLessThanOrEqual : "Package less than " + versionStringLessThan;
```